.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+
+1.  For large contributions or changes that include multiple files, [open an issue](#report-an-issue) and discuss it with us first. This may further prevent duplicate or unnecessary effort.
+
+1.  Familiarize yourself with the existing documentation. Look through and search the guides to decide where to add your topics.
+
+    -   The DevDocs team can find the best home for your new topics and add it to the navigation.
+    -   If a topic has a symlink, you can remove it with git commands and add a new file. Copy and paste a previous version of the topic to get started.
+
+1.  Focus on the content and on creating useful information for your fellow Magento developers and community members. Don't forget to review your work for typos, formatting errors, or sentences that need clarifying before opening a pull request.
+
+1.  Use the following guidelines to help you with the writing process:
+
+    -   Define the goal of your topic. What exactly do you want to teach the reader?
+    -   Make the title of your topic reflect the content.
+    -   Keep your sentences concise and try to separate conceptual information from procedural steps.
+    -   Remember to use active voice (not passive), write in the present tense, and use a friendly tone in second person. For example, _"You can now view the output..."_.
+    -   Use notes to alert readers about important details.
+    -   Use cross-references to other topics if appropriate. We can help you with the syntax if it is not clear. The template provides an example you can use.
+
+## Preview HTML locally
+
+To preview your changes in HTML output, follow the instructions in the [README](https://GitHub.com/magento/devdocs/blob/develop/README.md) to build the devdocs site locally using Jekyll.
+
+## Templates
+
+Edit and add content to existing pages. For new topics, we provide templates to help get you started:
+
+* [General topic template](https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/basic_template.html): This is a template for writing any topic with example formats and styles.
+* Tutorial templates: These templates provide example formats and styles for step-by-step instructions (like how-tos). Templates include a [first introduction topic](https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/tutorial-template-first.html), [middle topic](https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/tutorial-template-middle.html) for each step, and a [final step topic](https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/tutorial-template-last.html).
+
+### Edit metadata
+The Markdown (.md) file's metadata is a set of YAML key-value pairs. The metadata section is located at the top of each file.
+
+```yaml
+---
+group: install2
+title: Continue with your installation
+version: 2.1
+GitHub_link: install-gde/continue.md
+---
+```
+
+Refer to the following table for a description of each key-value pair.
```

Comment:
Maybe do a:

`> Key-value pair reference:`

Instead of an intro sentence to the table. I believe that adds a table title.

Reply:
Trying that out